OOP2/CoffeeMachine.java

```diff
@@ -0,0 +1,38 @@
+package OOP;
```

Comment:
I assume you added OOP2 folder by mistake ? This was part of the separate repo where I've already added comments ?